### PR TITLE
#167729384: improving-package-usability

### PR DESCRIPTION
**Pull request by sntakirutimana72**
#### What does this PR do?
  server file

- changing ES5 statements like (module.exports) to ES6 statements like (export default)

#### Description of Task to be completed?
- configured node to comply with ES6 statements
- no engine for web-pages just pure html pages
- express should use in-built method for body parsing
- And all other unnecessary routes removed as well.
- for cataloging sessions, I use express in-built session method
- removing jasmine and remain with only mocha as package for testing

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/167729384

#### Screenshots (if appropriate)
bookC

pack-json

serv

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env"
+  ]
+}
```
Comment (oesukam) [inline .babelrc]: New blank line required

File: server/routes/user.js [hunk 2]
```diff
@@ -87,5 +89,17 @@ router.get('/forgot', (req, res) => {
     res.redirect('/session');
   else res.render('forgot');
 });
+*/
 
-module.exports = router;
+const express = require('express');
```
Comment (oesukam) [inline server/routes/user.js]: @sntakirutimana72 Use ES6

File: server/routes/trip.js [hunk 3]
```diff
@@ -170,4 +173,16 @@ router.delete('/:trip_id', (req, res, next) => {
   } else return next('error');
 });
 
-module.exports = router;
+*/
+
+const express = require('express');
```
Comment (oesukam) [inline server/routes/trip.js]: @sntakirutimana72 Use ES6

File: .babelrc (hunk 1 above)
Comment (sntakirutimana72) [reply .babelrc]: I'm on it

**Comment by sntakirutimana72** [comment]
done all necessaries restructuring server.js file for LFA feedback.

**Pull request merged**